style-guide.md

```diff
@@ -11,59 +11,59 @@ Leave text in code blocks untranslated except for comments. You may optionally t
 Example:
 
 ```js
-// Example
+// Exemple
 import React from "react"
 export default () => (
-  <div style={{ color: `purple`, fontSize: `72px` }}>Hello Gatsby!</div>
+  <div style={{ color: `purple`, fontSize: `72px` }}>Salut Gatsby !</div>
```

Comment:
As it is the example (the reference) it should stay in English. My bad. I asked you to change this before.

Reply:
So change `// Example` to everything?

Reply:
No, only this first one

Reply:
Yup, it's fixed now.